.ci/run.sh

```diff
@@ -102,7 +102,6 @@ case "${CI_JOB}" in
 		sudo -E ln -sf "${config_path}/configuration-qemu.toml" "${config_path}/configuration.toml"
 		echo "INFO: Running qemu metrics tests"
 		sudo -E PATH="$PATH" ".ci/run_metrics_PR_ci.sh"
-		echo "INFO: Running cloud hypervisor metrics tests"
```

Comment:
why do you remove this?

Reply:
I think is useful to have it to mark the cloud hypervisor start

Reply:
Gaby, that info msg is located before the test starts, in the same way the msg for qemu does: https://github.com/kata-containers/tests/blob/f46c370026590300ad569f714728883a18a06933/.ci/run.sh#L114